website/content/docs/operators/deployment/schedulers/yarn.md

```diff
@@ -63,8 +73,19 @@ deployment on a multi-node YARN cluster.
 ### Topology Submission
 **Command**
 
+**Under 0.14.2 version (including 0.14.2)**
+
+`$ heron submit yarn heron-examples.jar com.twitter.heron.examples.AckingTopology AckingTopology`
+
+
+**After 0.14.3 version released**
+
 `$ heron submit yarn heron-examples.jar com.twitter.heron.examples.AckingTopology AckingTopology --extra-launch-classpath <extra-classpath-value>`
 
+>**Tips**
+>
+>More details for using the `--extra-launch-classpath` argument in 0.14.3 version. It supports both a single directory which including all `hadoop-lib-jars` and multiple directories separated by colon such as what `hadoop classpath` gives. ***What you should concern is the path you set must exists.***
```

Comment:
Should we replace `What you ... exists` with `The submit operation will fail if any path is invalid or if any file is missing.`